### PR TITLE
temperature response testing methods-update

### PR DESCRIPTION
**Pull request by joyvelasquez**
Updating the tests for the temperature response script as state by @wtbarnes in pull request https://github.com/HinodeXRT/xrtpy/pull/99.

### CODE REVIEW

File: xrtpy/response/tests/test_temperature_response.py [hunk 1]
```diff
@@ -70,7 +86,8 @@ def _IDL_temperature_response_raw_data(filename):
```
Comment (namurphy) [inline xrtpy/response/tests/test_temperature_response.py:87]:
```suggestion
assert filenames
```

File: xrtpy/response/tests/test_temperature_response.py [hunk 2]
```diff
@@ -1,23 +1,47 @@
...
+    assert path.exists()
+    filter_data_files = list(path.glob("**/*.txt"))
+    # import pdb; pdb.set_trace()
     return sorted(filter_data_files)
 
 
-filenames = get_IDL_data_files()
+filenames = get_IDL_data_files_older()
+
+# filenames = get_IDL_data_files()
```
Comment (joyvelasquez) [inline xrtpy/response/tests/test_temperature_response.py:44]:
```suggestion
```
Removing line if not used

**Comment by joyvelasquez** [comment]
Please refer to PR#150 titled 'Temperature Response Abundance Test' for the latest updates and address of the issue discussed in this PR and more. Consequently, I will be closing this PR."

**Comment by joyvelasquez** [comment]
Please refer to PR#150 titled 'Temperature Response Abundance Test' for the latest updates and address of the issue discussed in this PR and more. Consequently, I will be closing this PR."

**Pull request closed without merging**